conf/maestro-conf/template.devicedb.conf

```diff
@@ -212,17 +213,17 @@ tls:
     # key: path/to/key.pem
     
     # A PEM encoded 'client' type certificate
-    clientCertificate: {{SSL_CERTS_PATH}}/client.cert.pem
+    #clientCertificate: {{SSL_CERTS_PATH}}/client.cert.pem
```

Comment:
DeviceDB now connects as well now that identity.json is running before maestro starts up. Is there a need to remove the certificates?

Reply:
Removing TLS because fp-edge is handling it, not DeviceDB

Reply:
Please refer https://jira.arm.com/browse/PELEDGE19-1428 and https://jira.arm.com/browse/IOTCLOUDPR-2624
@psppalash